src/Uno.UI.Tests/DependencyProperty/Given_DependencyProperty.cs

```diff
@@ -1815,6 +1815,17 @@ public void When_DefaultValueOverride()
 			Assert.AreEqual(42, SUT.GetValue(MyDependencyObjectWithDefaultValueOverride.MyPropertyProperty));
 		}
 
+		[TestMethod]
+		public void ClearValue()
+		{
+			var SUT = new MainPage();
+			Assert.AreEqual(true, SUT.MyCheckBox.IsChecked);
+			Assert.AreEqual(true, SUT.MyModel.IsChecked);
+			SUT.MyButton.RaiseClick();
+			Assert.AreEqual(false, SUT.MyCheckBox.IsChecked);
+			Assert.AreEqual(true, SUT.MyModel.IsChecked);
+		}
```

Comment:
It's probably best to make `DependencyObject`-specific tests, at least one for ClearValue and one for bindings clearing. You could probably move this new test to Button or FrameworkElement related test suites.

Reply:
> It's probably best to make DependencyObject-specific tests, at least one for ClearValue and one for bindings clearing.

@jeromelaban Could you elaborate more on this please? Thanks